README.md

```diff
@@ -4,6 +4,77 @@ JS Client library to interact with Supabase Storage.
 
 - Documentation: https://supabase.io/docs/reference/javascript/storage-createbucket
 
+## Quick Start Guide
+
+### Installing the module
+
+``` bash
+npm install @supabase/storage-js
+```
+
+### Connecting to the storage backend
+
+``` bash
+# .env
+SUPABASE_STORAGE_URL=https://<project_ref>.supabase.co/storage/v1
+SUPABASE_SERVICE_KEY=<service_role>
+```
+
+``` js
+// index.hs
+import { SupabaseStorageClient } from '@supabase/storage-js'
+import 'dotenv/config'
+
+const STORAGE_URL = process.env.SUPABASE_STORAGE_URL;
+const SERVICE_KEY = process.env.SUPABASE_SERVICE_KEY; //! service key, not anon key
+
+const storage_client = new SupabaseStorageClient(
+    STORAGE_URL,
```

Comment:
Nit: indentation of 2 spaces for consistency with documentation elsewhere?

Reply:
Will solve this tonight when i get home!

Reply:
[Done](https://github.com/supabase/storage-js/pull/47/commits/2aa63b7bc3624b691912300111a7028f00dd2c96)